client/components/web-preview/toolbar.jsx

```diff
@@ -141,12 +133,7 @@ PreviewToolbar.defaultProps = {
 };
 
 const mapStateToProps = ( state, ownProps ) => {
-	const site = getSelectedSite( state );
-	const showSEO = ownProps.showSEO && (
-		( site && site.plan && hasBusinessPlan( site.plan ) && isEnabled( 'manage/advanced-seo' ) ) ||
```

Comment:
This is taking away the business plan check too. Are we sure that we want to do this since this feature is limited to business plan subscribers?